build/build-and-unit-tests.yml

```diff
@@ -9,8 +9,16 @@ jobs:
 - job: BuildAndUnitTests${{ parameters.configuration }}
   displayName: Build and run unit tests - ${{ parameters.configuration }}
```

Comment:
Not sure why, but the DisplayName is getting ignored in the pipeline output

Reply:
Updated the new pipeline link.